app/src/main/java/org/simple/clinic/editpatient/EditPatientModel.kt

```diff
@@ -29,7 +30,8 @@ data class EditPatientModel(
         phoneNumber: PatientPhoneNumber?,
         dateOfBirthFormatter: DateTimeFormatter,
         bangladeshNationalId: BusinessId?,
-        saveButtonState: EditPatientState?
+        saveButtonState: EditPatientState?,
+        colonyOrVillagesList: List<String>?
```

Comment:
Since we won't be passing the list when creating the model using this factory function we can remove this from the function param, WDYT?